renovate.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
paging @lfrancke or @teozkr as the resident Renovate experts to have a look :)

Reply:
But would't it replace the tag with the digest and then add the tag in a comment at the end like it does with github actions?

https://github.com/stackabletech/operator-templating/pull/161/files

Reply:
Let me go check...

Reply:
https://docs.renovatebot.com/docker/#digest-updating implies that it will try to update both, but I'm not sure about to which degree this works to for third-party registries (especially "weird" ones that don't use Docker's own registry server).

I guess we could try running it locally and see what it tries to do...

Reply:
Further [down](https://docs.renovatebot.com/docker/#digest-updating)) it sounds like it would not update though..

> To keep things simple, Renovate keeps the Docker tag in the FROM line, like this: FROM node:14.15.1@sha256:d938c1761e3afbae9242848ffbb95b9cc1cb0a24d889f8bd955204d347a7266e.
> 
> Digest Updating
> If you follow our advice to replace a simple tag like node:14 with a pinned digest node:14@sha256:d938c1761e3afbae9242848ffbb95b9cc1cb0a24d889f8bd955204d347a7266e, you will get Renovate PRs whenever the node:14 image is updated on Docker Hub.